src/toggle/toggle.component.ts

```diff
@@ -102,6 +102,16 @@ export class Toggle extends Checkbox {
 	 */
 	static toggleCount = 0;
 
+	/**
+	 * Variable set the value to input true/false.
+	 */
+	value = false;
+
+	/**
+	 * Variable set the name to input same as Id.
+	 */
+	name = "toggle-" + Toggle.toggleCount;
+
```

Comment:
`value` & `name` are metadata (@Input) in Checkbox and are inherited. They appear as `undefined` because they are not initialized in checkbox component.

You can instead change the template (line 70 & 71) to the following:

![image](https://github.com/carbon-design-system/carbon-components-angular/assets/38994122/9486efdd-5353-4aaf-a0bd-7a617f9eba11)


```
			[attr.value]="value"
			[attr.name]="name"
```

This way, if they equal undefined or null, they will be removed from the element.

Reply:
The below change will be unnecessary after & can be removed.

Reply:
ok removed